.github/scripts/map-affected-files-to-modules.sh

```diff
@@ -21,30 +21,40 @@ echo "Found modules: $modules"
 declare -A unique_modules
 
 for path_to_file in $changed_files; do
-	echo "Resolving a module affected by a file: '$path_to_file'"
```

Comment:
???

Reply:
idk why that file got changed but it's fixed now.